.activate-venv.sh

```diff
@@ -13,12 +13,18 @@ else
     echo "* the no-delete-venv flag is set. preserving venv"
 fi
 
-if [ ! -d venv ]; then
-    # build venv if one doesn't exist
-    virtualenv --python=`which python2` venv
+python=/usr/bin/python3.5
```

Comment:
Alfred seems to have no problem with Python 3 https://github.com/elifesciences/builder-private/blob/master/salt/top.sls#L263